doc/API.md

```diff
@@ -68,3 +88,41 @@ filtered by an optional `status` parameter.
 | Field         | Type                                       | Description               |
 | ------------- | ------------------------------------------ | ------------------------- |
 | `vaults`      | array of [vault resource](#vault-resource) | Vaults filtered by status |
+
+### `sigvault`
+
+The `sigvault` RPC Command executes the signing process of the Revault
+pre-signed transactions.
+
+#### Request
+
+| Parameter        | Type   | Description                         |
+| ---------------- | ------ | ----------------------------------- |
+| `vault_id`       | string | Unique id of the vault to sign      |
+| `only_emergency` | bool   | Sign only the emergency transaction |
+
+#### Response
+
+TODO: specify response
+
+### `spendvault`
+
+The `spendvault` RPC command creates a new spending request and creates
+a [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md) 
+spending the vault with the given ID and send it to cosigning
+peers.
+
+#### Request
+
+| Parameter  | Type   | Description                         |
+| ---------- | ------ | ----------------------------------- |
+| `vault_id` | string | Unique id of the vault to spend     |
+
+TODO: add parameters
+
+#### Response
+
+| Field        | Type   | Description                      |
+| ------------ | ------ | -------------------------------- |
+| `psbt`       | string | The partially signed transaction |
+| `request_id` | string | The spending request ID          |
```

Comment:
Actually it's going to be more complex. On the top of my head you need to at least get the fee-bumping PSBT and spend PSBT.